### ISSUE TITLE
Getting error from AzureOpenAI endpoint: logprobs, best_of and echo parameters are not available on gpt-35-turbo model

### ISSUE THREAD

**Issue by hario90** [microsoft/semantic-kernel#1564]
**Describe the bug**
Getting 400 HTTP status code response from AzureOpenAI when serviceId and deploymentOrModelId are set to "gpt-35-turbo".

```

fail: Microsoft.SemanticKernel.IKernel[0]
      Something went wrong while rendering the semantic function or while executing the text completion. Function: FunSkill.Joke. Error: Invalid request: The request is not valid, HTTP status: 400. Details: logprobs, best_of and echo parameters are not available on gpt-35-turbo model. Please remove the parameter and try again. For more details, see https://go.microsoft.com/fwlink/?linkid=2227346.
      Status: 400 (BadRequest)
      ErrorCode: BadRequest

      Content:
      {"error":{"code":"BadRequest","message":"logprobs, best_of and echo parameters are not available on gpt-35-turbo model. Please remove the parameter and try again. For more details, see https://go.microsoft.com/fwlink/?linkid=2227346."}}
```

**To Reproduce**
1. Clone https://github.com/microsoft/semantic-kernel-starters/tree/main/sk-csharp-console-chat
2. In terminal, change directory to sk-csharp-hello-world
3. follow README.md instructions in this directory to set dotnet secrets for serviceId and deploymentOrModelId to "gpt-35-turbo"
4. In terminal, run

```
dotnet build
dotnet run
```

**Expected behavior**
Console app runs without errors.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - IDE: N/A
 - NuGet Package Version: 0.15.230531.5-preview and 0.16.230615.1-preview (reproducible in both)

**Additional context**

**Comment by hario90** [comment]
Closing since I was using a ChatCompletion model when the EndpointType in KernelSettings was set to TextCompletion.

**Comment by ratdoux** [comment]
Arghh thank you!!!

**Comment by CalvinQuark** [comment]
I encountered this issue while intentionally targeting the `gpt-35-turbo` model. This model is not one indicated in the [Hello World example's README](https://github.com/microsoft/semantic-kernel-starters/blob/main/sk-csharp-hello-world/README.md) file which instead specifies that the `text-davinci-003` model is to be targeted.

When I created a new _Azure OpenAI_ resource, I requested the full gamut of capabilities in the approval request step. Nevertheless, the `text-davinci-003` model is not deployable in my Azure AI Studio instance. The Studio portal indicates "No" in the _Deployable_ column and provides a tooltip stating "This model must be fine-tuned before it can be deployed". But the Studio doesn't appear to either include or link to instructions regarding how the model can be fine-tuned and therefore deployed. Hence, the onboarding experience is a bit confusing to the uninitiated.
